src/main/java/core/basesyntax/impl/StorageImpl.java

```
@@ -33,21 +23,24 @@ public void setValue(V[] value) {
 
     @Override
     public void put(K key, V value) {
+        if (size >= MAX_ITEMS_NUMBER) {
+            return;
```

let's throw an exception. because this looks like data leak